simulator/snapshot/snapshot.go

```diff
@@ -42,6 +42,7 @@ type ResourcesForSnap struct {
 
 // ResourcesForLoad indicates all resources and scheduler configuration to be loaded.
 type ResourcesForLoad struct {
+	Sas             []v1.ServiceAccountApplyConfiguration             `json:"sas"`
```

Comment:
Now `SnapshotService` is only used by `SnapshotHandler` and it only handles resources of the fake cluster. So we don't need to change `SnapshotService`.

Reply:
Do I need to delete this line?

Reply:
There will be an exception after deletion

![image](https://github.com/user-attachments/assets/bb0d123e-1d20-4c5c-a1f4-8a99ceef40f7)

Reply:
I mean we don't need the entire fix in `SnapshotService`, not only the line.

Reply:
> I mean we don't need the entire fix in `SnapshotService`, not only the line.

Wow, I seem to be losing my mind, I don't understand your question at all

Reply:
First, #421 occurs when you use [oneshotimporter and syncer](https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/master/simulator/docs/import-cluster-resources.md), doesn't it? And these features don't depend on `SnapshotService`. So in my understanding, #421 isn't related to `SnapshotService`.
If it's wrong, please tell me why we need to edit `SnapshotService`.

Reply:
> Almost the same as `Load`. You would add `Sas` to `ResourcesForSnap`, and implement `listSas`.

done